packages/template-retail-react-app/app/components/footer/index.jsx

```diff
@@ -30,7 +29,8 @@ import {HideOnDesktop, HideOnMobile} from '../responsive'
 import {getPathWithLocale} from '../../utils/url'
 import LocaleText from '../locale-text'
 import useMultiSite from '../../hooks/use-multi-site'
-
+import {isServer} from '../../utils/utils'
+const [StylesProvider, useStyles] = createStylesContext('Footer')
```

Comment:
StylesProvider imported from chakra/react is deprecated, we need to use `createStylesContext`